frontend/src/pages/OrganizationProfile.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { useEffect, useState } from 'react';
+import { OrderDirection } from '../common/enums/order-direction.enum';
+import { IPaginatedEntity } from '../common/interfaces/paginated-entity.interface';
+import Divisions, { DivisionTabs, DivisionType, IDivision } from '../components/Division';
+import { useErrorToast } from '../hooks/useToast';
+import PageLayout from '../layouts/PageLayout';
+import i18n from '../common/config/i18n';
+import { SortOrder, TableColumn } from 'react-data-table-component';
+import { useDivisionDataQuery } from '../services/division/division.service';
+
+const OrganizationProfile = () => {
+  const [divisionType, setDivisionType] = useState<DivisionType>(DivisionType.Branches);
+  const [data, setData] = useState<IPaginatedEntity<IDivision>>();
+  const [page, setPage] = useState<number>();
+  const [rowsPerPage, setRowsPerPage] = useState<number>();
+  const [orderByColumn, setOrderByColumn] = useState<string>();
+  const [orderDirection, setOrderDirection] = useState<OrderDirection>();
+
+  const {
+    data: divisionData,
+    isLoading,
+    error,
+  } = useDivisionDataQuery(
+    rowsPerPage as number,
+    page as number,
+    orderByColumn as string,
+    orderDirection as OrderDirection,
+    divisionType as DivisionType,
+  );
+
+  useEffect(() => {
+    if (divisionData?.meta) {
+      setPage(divisionData.meta.currentPage);
+      setRowsPerPage(divisionData.meta.itemsPerPage);
+      setOrderByColumn(divisionData.meta.orderByColumn);
+      setOrderDirection(divisionData.meta.orderDirection);
+    }
+  }, []);
+
+  useEffect(() => {
+    if (error) useErrorToast(i18n.t('general:error.load_entries'));
+  }, [error]);
+
+  const handleTabClick = (id: number) => {
```

Comment:
We should keep the same format as for the methods below, since this is still and event we can call it onTabClick